recipes/python-drmaa/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7.6" %}
+
+package:
+  name: python-drmaa
+  version: {{ version }}
+
+source:
+  fn: drmaa-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/d/drmaa/drmaa-{{ version }}.tar.gz
+  md5: 3cb6dade683d0621fb14871ef1261f9b
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  files:
+    - run_test.py
```

Comment:
Is it ok if I leave it though? I kind of like the visibility we are giving it in the `meta.yaml`. If you don't like it still, I can of course remove it.